### PR TITLE
Add guess tests

### PR DESCRIPTION
**Pull request by JorjaF**
so i added the tests for the guess class, can you take a look at them before I start coding, please?

### CODE REVIEW

File: spec/guess_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+require './lib/card'
+require './lib/guess'
+require 'rspec'
+
+RSpec.describe Guess do
+
+  it "exists" do
+    card = Card.new("10", "Hearts")
+    guess = Guess.new("10 of Hearts", card)
+    expect(guess).to be_instance_of(Guess)
```
Comment (dcoleman21) [inline spec/guess_spec.rb:10]: You can add an expect here for `card` as well if you would like

File: spec/guess_spec.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  end
+   
+  it 'can make a correct guess' do 
+    card = Card.new("10", "Hearts")
+    guess = Guess.new("10 of Hearts", card)
+    guess.card
```
Comment (dcoleman21) [inline spec/guess_spec.rb]: I would wrap `guess.card` in an expect
```ruby
expect(guess.card).to eq(card)
```

File: spec/guess_spec.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+RSpec.describe Guess do
+
+  it "exists" do
+    card = Card.new("10", "Hearts")
+    guess = Guess.new("10 of Hearts", card)
+    expect(guess).to be_instance_of(Guess)
+  end
+   
+  it 'can make a correct guess' do
```
Comment (dcoleman21) [inline spec/guess_spec.rb]: Make this an `"it has attributes" do` test

File: spec/guess_spec.rb [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+   
+  it 'can make a correct guess' do 
+    card = Card.new("10", "Hearts")
+    guess = Guess.new("10 of Hearts", card)
+    guess.card
+    expect(guess.response).to eq("10 of Hearts")
+    expect(guess.feedback).to eq("Correct!")
+  end 
+
+  it "can have an incorrect guess" do
```
Comment (dcoleman21) [inline spec/guess_spec.rb]: You can put both the `correct` and `incorrect` guess in the same test. 
So you would want a test here that calls on `guess.correct?` which should return `true` or `false`. So you can set the test up to return both of those.

File: spec/guess_spec.rb [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    card = Card.new("Queen", "Clubs")
+    guess = Guess.new("2 of Diamonds", card)
+    guess.card
+    expect(guess.response).to eq("2 of Diamonds")
+    expect(guess.feedback).to eq("Incorrect!")
```
Comment (dcoleman21) [inline spec/guess_spec.rb]: Then I would write a test 
```ruby
it 'can give feedback for a guess' do
```
Where you are testing `guess.feedback` and returning both `"Correct"` and `"Incorrect"`.

**Review by dcoleman21 — CHANGES_REQUESTED** [review]
Take a look at the comments that I have left for you. Let me know what questions you have and/or bring them to your checkin tomorrow.

**Review by JorjaF — COMMENTED** [review]
hey i made some changes to the guess tests, can you double check them for me please?

File: spec/guess_spec.rb (hunk 2 above)
Comment (JorjaF) [reply spec/guess_spec.rb]: updated

**Pull request merged**